XCUITests/HistoryTests.swift

```diff
@@ -163,7 +163,8 @@ class HistoryTests: BaseTestCase {
         // 'x' button to close the tab is not visible, so closing by swiping the tab
         app.collectionViews.cells[webpage["label"]!].swipeRight()
 
-        navigator.goto(HomePanelsScreen)
+        navigator.performAction(Action.OpenNewTabFromTabTray)
```

Comment:
looks good, but also we should check whether we can fix the timing issue that's happening when we go to HomePanelsScreen directly.

Reply:
The solution is that or add wait for elements and nowAt so that we are sure we are on the HomePanelScreen, thought this solution looks better and it will be more consistent